src/sentry/incidents/logic.py

```diff
@@ -1243,6 +1243,7 @@ def get_alertable_sentry_apps(organization_id, with_metric_alerts=False):
         installations__organization_id=organization_id,
         is_alertable=True,
         installations__status=SentryAppInstallationStatus.INSTALLED,
+        installations__date_deleted=None,
```

Comment:
@scefali not too familiar with the problem, but if `installations__status` is installed, shouldn't `installations__date_deleted` be None anyways. They seem to be of the same type of filter.

Reply:
@NisanthanNanthakumar When a sentry app is uninstalled, we don't change the status. We just set `date_deleted` to the current time. This is the case we are checking for.

Reply:
@scefali 👍 I guess this is okay for now, but it's weird that we don't change the installation status.  Maybe it can be a separate PR